distributed/worker.py

```diff
@@ -2229,13 +2231,21 @@ def trigger_profile(self):
             active_threads = self.active_threads.copy()
         frames = sys._current_frames()
         frames = {ident: frames[ident] for ident in active_threads}
+        llframes = {}
+        if self.low_level_profiler:
+            llframes = {ident: profile.ll_get_stack(ident) for ident in active_threads}
         for ident, frame in frames.items():
             if frame is not None:
                 key = key_split(active_threads[ident])
-                profile.process(frame, None, self.profile_recent,
-                                stop='distributed/worker.py')
-                profile.process(frame, None, self.profile_keys[key],
+                init = profile.create()
```

Comment:
This doesn't seem to be used

Reply:
You mean the `init` variable only, right? I think we still need the `profile.create()`

Reply:
I think that we use `self.profile_recent` as the state object here instead.  We don't need to keep creating a new state I don't think

Reply:
I'll double-check that then, you may be right.

Reply:
Removing this, you were right in your comment @mrocklin. Thanks for noticing that.